src/interface/file.rs

```diff
@@ -321,8 +321,8 @@ impl EmulatedFileMap {
         let new_mapsize = self.mapsize + MAP_1MB;
         let _newsize = f.set_len((COUNTMAPSIZE + new_mapsize) as u64).unwrap();
 
-        let newmap : Vec::<u8>;
-        let newcountmap : Vec::<u8>;
+        let newmap : Vec<u8>;
+        let newcountmap : Vec<u8>;
```

Comment:
Same argument here.  Let's not make this edit for now.  Defer after discussions of [another issue](https://github.com/Lind-Project/lind_project/issues/373).